### PR TITLE
Issue 2927

### LINKED ISSUE #2927: [BUG] Balance row text alignment
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate problems

### Describe the bug

Just tried a recent nightly build and the first thing I noticed was a slight value misplacement for the balancerow strings.

### To Reproduce

_No response_

### Expected behavior

Vertically align the value and currency code in the BalanceRow.

### Screenshots

67b8a09c-8a89-45df-9234-dd5fc586a335

### Smartphone

Galaxy S23

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sandeepjak2007**
Fixed issue 2927

## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read the [Architecture Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Architecture.md).
- [x] I confirm that I've run the code locally and everything works as expected.

## Does this PR close any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #2927 

> Replace `2927` with the number of the GitHub issue that this PR is fixing. If you've done that correctly, you'll see the issue title linked when previewing your PR description.

> **!Note:** Do **not** link the PR number. Link the number/id of the GitHub Issue from [issues](https://github.com/Ivy-Apps/ivy-wallet/issues).

## Troubleshooting CI failures ❌

GitHub Actions failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

File: screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt [hunk 1]
```diff
@@ -110,20 +110,12 @@ private fun BoxWithConstraintsScope.UI(
...
-
-                    Text(
-                        text = state.totalBalanceWithoutExcludedText,
-                        style = UI.typo.nC.style(
-                            color = Gray,
-                            fontWeight = FontWeight.Bold
-                        )
+                    IncomeExpensesRow(
+                        currency = state.baseCurrency,
+                        incomeLabel = "Total Balance",
```
Comment (ILIYANGERMANOV) [inline screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt:115]: This string should be extracted as a string resource so the app can be translated

**Review by ILIYANGERMANOV — COMMENTED** [review]
LGTM! Can you please extract string resources? You'll like get Lint translation error - if that happens, just copy-paste the English string in all translations

**Comment by ILIYANGERMANOV** [comment]
@sandeepjak2007 before merging can you also please attach screenshots of the UI in Light & Dark mode? That's a requirement for PRs because I don't have time to manually test it

File: screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt [hunk 2]
```diff
@@ -110,20 +110,12 @@ private fun BoxWithConstraintsScope.UI(
...
+                        income = state.totalBalanceWithoutExcluded.toDouble(),
+                        expensesLabel = "Total Balance ${stringResource(id = R.string.excluded)}",
```
[the remark this answers is not in the document]
Comment (sandeepjak2007) [reply screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt]: @ILIYANGERMANOV 

I have committed the string resources please check and let me know if any changes to bee made.

Thanks

**Comment by sandeepjak2007** [comment]
Sure will Add the screenshot

**Review by ILIYANGERMANOV — APPROVED** [review]
Fix the Detekt errors and we can merge

File: screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt [hunk 3]
```diff
@@ -110,20 +110,12 @@ private fun BoxWithConstraintsScope.UI(
...
-                        text = state.totalBalanceWithoutExcludedText,
-                        style = UI.typo.nC.style(
-                            color = Gray,
-                            fontWeight = FontWeight.Bold
-                        )
+                    IncomeExpensesRow(
+                        currency = state.baseCurrency,
+                        incomeLabel = stringResource(id = R.string.total_balance),
+                        income = state.totalBalanceWithoutExcluded.toDouble(),
+                        expensesLabel = "${stringResource(id = R.string.total_balance)} ${stringResource(id = R.string.excluded)}",
```
[the remark this answers is not in the document]
Comment (ILIYANGERMANOV) [reply screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt:117]: @sandeepjak2007 also please fix this one, too

[the remark this answers is not in the document]
Comment (sandeepjak2007) [reply screen/accounts/src/main/java/com/ivy/accounts/AccountsTab.kt]: Changed.

**Comment by sandeepjak2007** [comment]
**Dark Theme**
ivy-dark

**Light Theme**
Lightmode

**Review by ILIYANGERMANOV — APPROVED** [review]
Thanks for making the changes!

**Comment by ILIYANGERMANOV** [comment]
> @sandeepjak2007, since you marked this PR as a solution to my issue, could you please try to align the BalanceRow items (1,200 is a little lower than USD on the Revolut card in your screenshot)?
> 
> Also I'd make a little more margin under "Accounts" label. But it indeed looks cool now, thx.

+1 for both 👍 @sandeepjak2007 can you make those improvements 🙏

**Comment by sandeepjak2007** [comment]
> > @sandeepjak2007, since you marked this PR as a solution to my issue, could you please try to align the BalanceRow items (1,200 is a little lower than USD on the Revolut card in your screenshot)?
> > Also I'd make a little more margin under "Accounts" label. But it indeed looks cool now, thx.
> 
> +1 for both 👍 @sandeepjak2007 can you make those improvements 🙏

Sure will make changes and share the screenshot. Once review and push changes then.

**Comment by sandeepjak2007** [comment]
@nvllz @ILIYANGERMANOV  Can you please review the view once
Ivy3

**Comment by ILIYANGERMANOV** [comment]
I'm still not sure if "1,200 USD" is aligned. "USD" is a bit above 1,200 but that's not easily fixable - will merge the PR cuz it's an improvement. @nvllz we can re-open your issue if it needs more work 👍

**Comment by sandeepjak2007** [comment]
Big thanks to @ILIYANGERMANOV & @nvllz  for your patience and guidance in assisting me with my initial contributions to an open-source project, specifically with design changes.

Thanks a lot.

**Pull request merged**